execution/sym_region_frag_machine.ml

```diff
@@ -668,6 +668,9 @@ struct
 	let cbase = List.fold_left Int64.add 0L cbases in
 	let (base, off_syms) = match (cbase, syms, ambig) with
 	  | (0L, [], []) -> raise NullDereference
+	  (* This is the case to use when applying table treatment for symbolic
```

Comment:
It seems to me like this comment could apply equally well to the (0L, [v], _) case, right?